src/components/button/button.svelte

```diff
@@ -294,15 +294,25 @@
     &:disabled:not(.isLoading) {
       --color: var(--leo-color-text-primary);
     }
+
+    &.fab {
+      --padding-y: 0;
+      --padding-x: 0;
+    }
   }
   .leoButton.isPlainFaint {
     --radius: 8px;
     --padding-x: 2px;
-    --color: var(--leo-color-text-secondary);
+    --color: var(--leo-button-faint-color, var(--leo-color-text-secondary));
     --color-hover: var(--leo-color-gray-70);
     --box-shadow-hover: none;
-    --icon-color: var(--leo-color-icon-default);
+    --icon-color: var(--leo-button-faint-color, var(--leo-color-icon-default));
```

Comment:
The apparent intent of the plain-faint button is that it _can_ use the same color as the surrounding text. This adds a variable to set that color.

Reply:
Discussed that we should wait to add this until https://github.com/brave/leo/pull/490 lands.